File: insert.php
<?php

$m = new MongoDB\Driver\Manager("mongodb://localhost:27017");


echo "Connection to database successfully <br>";
$bulkWrite = new MongoDB\Driver\BulkWrite;
$doc = ['name' => 'Deepak', 'class' => 'TYIT', 'Rollno' => '229749'];
$bulkWrite->insert($doc);

$m->executeBulkWrite('TYITDB1.TYITCOL', $bulkWrite);


echo "Data inserted successfully";













// $m->MongoClient();
// $db = $m->selectDB('TYITDB1');
// $collection = $db->selectCollection('TYITCOL');
// $document = array(
//     "name" => "Deepak",
//     "age" => 23,
//     "RollNo" => 229749,
//     "Project" => "Guess what?"

// );

//Insert Document into the collection



// $collection->insert($document);
// echo "Data inserted successfully";
 ?>

File: insert.py
from pymongo import MongoClient
client = MongoClient('localhost', 27017)
db = client.TYITDB1
def insert():
    try:
        name = input("Enter the name:")
        Rollno = int(input("Enter rollno:"))
        Age = int(input("Enter age:"))
        id = int(input("Enter id:"))
        db.TYITCOL.insert_one({
            "name": name,
            "Rollno": Rollno,
            "Age": Age,
            "id": id
        })
        print("Data inserted successfully!")
    except Exception as e:
        print(str(e))


insert()
